core/services/webhook/external_initiator_manager.go

```diff
@@ -124,7 +159,8 @@ func (m externalInitiatorManager) DeleteJob(webhookSpecID int32) error {
 
 func (m externalInitiatorManager) FindExternalInitiatorByName(name string) (bridges.ExternalInitiator, error) {
 	var exi bridges.ExternalInitiator
-	return exi, m.db.First(&exi, "lower(name) = lower(?)", name).Error
+	err := m.db.Get(&exi, "SELECT * FROM external_initiators WHERE lower(external_initiators.name) = lower($1)", name)
```

Comment:
Use `postgres.Q` here.

Pretty sure I made this comment before but it seems to have disappeared 🤔